hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsRestOperation.java

```diff
@@ -121,7 +121,15 @@ public AbfsHttpOperation getResult() {
    * HTTP operations.
    */
   void execute() throws AzureBlobFileSystemException {
+    // see if we have latency reports from the previous requests
+    String latencyHeader = this.client.getAbfsPerfTracker().getClientLatency();
+
+    if (latencyHeader != null && !latencyHeader.isEmpty()) {
+      requestHeaders.add(new AbfsHttpHeader("x-ms-abfs-client-latency", latencyHeader));
```

Comment:
String constants? Extract the header too.

Reply:
Using string constants now. Extracted the header.